starter-code/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <title>Blog</title>
     <!-- TODO: Add a meta tag to properly set the viewport and initial-scale -->
+    <meta name="viewport" content="width=divice-width, initial-scale=1">
```

Comment:
Typo here. Be sure to get in the habit of working with your Chrome Developer Tools console window open. It will often let you know you've got an error, and which line it's on.